src/BinSkim.Sdk/BinaryAnalyzerContext.cs

```diff
@@ -135,6 +135,6 @@ public void Dispose()
                 "A shared CompilerDataLogger instance that will be passed to all skimmers.");
 
 
-        public int MaxFileSizeInKilobytes { get; set; }
+        public long MaxFileSizeInKilobytes { get; set; }
```

Comment:
Synced to the latest and met the following build error.
<img width="734" alt="image" src="https://user-images.githubusercontent.com/119978670/209993217-e5f95bc2-4a75-4fa1-a3a1-7dd5ef4a96ed.png">